taverna-docker-activity/pom.xml

```diff
@@ -108,19 +124,14 @@
 			<groupId>org.apache.taverna.engine</groupId>
 			<artifactId>taverna-workflowmodel-impl</artifactId>
 			<version>${taverna.engine.version}</version>
-			<scope>test</scope>
```

Comment:
Is `taverna-workflowmodel-impl` needed at compile time? If not, re-add the <scope>